cdk/custom-auth/magic-link.ts

```diff
@@ -43,19 +43,33 @@ import {
 } from "./common.js";
 
 let config = {
+  /** Should Magic Link sign-in be enabled? */
```

Comment:
It's unclear to me what exactly to do here.

Reply:
Updated the comment